src/controllers/webhooks.js

```diff
@@ -60,7 +62,7 @@ function computeUpdatedPrice(event) {
 */
 function orderChanges(incomingOrder, existingOrder) {
   // this function will return an object of updated values relevant to our DB table
-  const { note_attributes: noteAttributes, updated_at: updatedAt, donation_amount: donationAmount } = incomingOrder;
+  const { note_attributes: noteAttributes, updated_at: updatedAt, total_tip_received: donationAmount } = incomingOrder;
```

Comment:
this was causing donationAmount to get parsed incorrectly as `NaN`, 

but I also decided to implement this `parseFloatSafe` function anyways to be extra safe?